<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_AdvancedBase</class>
 <widget class="QFrame" name="PropertyWidget_AdvancedBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="textBaseLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ScrSpinBox" name="textBase">
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="trackingLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ScrSpinBox" name="tracking"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="scaleHLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScrSpinBox" name="scaleH"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="scaleVLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="ScrSpinBox" name="scaleV"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="wordTrackingLabel">
     <property name="text">
      <string>Word Spacing</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="minWordTrackingLabel">
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ScrSpinBox" name="minWordTrackingSpinBox">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="ScrSpinBox" name="normWordTrackingSpinBox">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="normWordTrackingLabel">
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="glyphExtensionLabel">
     <property name="text">
      <string>Glyph Extension</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="minGlyphExtensionLabel">
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ScrSpinBox" name="minGlyphExtSpinBox">
     <property name="minimum">
      <double>90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>110.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="ScrSpinBox" name="maxGlyphExtSpinBox">
     <property name="minimum">
      <double>90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>110.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="maxGlyphExtensionLabel">
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
